### ISSUE TITLE
Marking sensitive content and such

### ISSUE THREAD

**Issue by vagabondsun** [jakeledoux/crabber#38]
as an adult, i can imagine that sometimes i might want to talk about Adult Things on crabber. but given that i'm pretty sure there are also minors on the site, and _especially_ since you can indiscriminately view a feed of literally everyone's molts, i think it's pretty imperative that there be some kind of feature(s) to make sure kids aren't seeing someone's lewd posts. this could be any (or all?) of the following:

- an option to hide your posts from the wild west (either per-post or for the whole profile)
- a 'sensitive content' toggle (again, per-post and/or for the whole profile)
- the ability to make your posts or profile visible to followers only

twitter has the latter of these two features already, so i think it also makes sense to have them on that front as well.

**Comment by jakeledoux** [comment]
Right. This wouldn't be super difficult to implement, just really tedious. Gonna have to update all of the molt queries to filter for NSFW preferences as well as figure out UI for marking Molts as NSFW.

@xpyxel Let me know if you're interested in doing the backend stuff here I'll get started on the UI. Otherwise I'll take care of it.
